api/src/api/session.js

```diff
@@ -30,6 +30,46 @@ router.get(
   }),
 );
 
+//get all displayed sessions on the home page
+router.get(
+  '/displayed',
+  errorWrap(async (req, res) => {
+    let activeSessions = await Session.find({ active: 'true' });
+    if (activeSessions.length >= 10) {
+      res.status(200).json({
+        message: `Successfully retrieved all displayed sessions on the home page.`,
+        success: true,
+        result: activeSessions,
+      });
+      return;
+    }
+
+    const inactiveSessions = await Session.find({ active: 'false' }).sort({
+      startTime: 'asc',
```

Comment:
I think this should be just "1" or "-1" for ascending or descending. Not 100% sure though so make sure to look it up

Reply:
Ah I went over this with Albert Thursday and he recommended 'asc'. [Here'](https://stackoverflow.com/questions/4299991/how-to-sort-in-mongoose)s where I got it from.

Reply:
Ohh my bad. Trust albert then 😎